src/test/data/JsonDengueHotspotTrackerStorageTest/invalidPersonDengueHotspotTracker.json

```
@@ -3,6 +3,7 @@
     "name": "Person with invalid name field: Ha!ns Mu@ster",
     "postal": "948242",
     "date": "2023-03-05",
-    "age": "23"
+    "age": "23",
+    "tag": "DENV1"
```

Is the tag format invalid on purpose?

```suggestion
    "tagged": [ "DENV1" ]
```

---

Oops, looks like I missed the brackets around the tags!! Thanks for catching that